HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -621,6 +622,7 @@ public IActionResult Confirmation_Get(bool emailPreferenceSubmitted)
         {
             ReferenceCode = questionnaire.ReferralCode,
             LocalAuthorityName = questionnaire.LocalAuthorityName,
+            LocalAuthorityMessagePartialLocation = GetLocalAuthorityConfirmationMessagePartialViewPath(questionnaire),

```

Comment:
Both the above properties should be updated to `…PartialViewPath`, instead of `…PartialLocation`.